daal4py/sklearn/cluster/_k_means_0_23.py

```diff
@@ -33,7 +33,7 @@
 from sklearn.cluster import KMeans as KMeans_original
 
 import daal4py
-from .._utils import getFPType, method_uses_sklearn, method_uses_daal
+from .._utils import getFPType, method_uses_sklearn, method_uses_daal, daal_check_version
```

Comment:
Do we need update interfaces for 0.21 and 0.22 Sklearn version? Does it correct work if we take the newest version of daal and old version d4p (do we need check version for dp4?)